packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -260,7 +259,8 @@ export class GCFBootstrapper {
           // TODO: find out the best way to get this type, and whether we can
           // keep using a custom event name.
           await this.probot.receive({
-            name: name as EventNames.StringNames,
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            name: name as any,
```

Comment:
a type seemed to also be randomly deleted from `Event.StringNames`, didn't have the cycles to debug this right now.